src/aind_behavior_experiment_launcher/behavior_launcher/slims_picker.py

```diff
@@ -0,0 +1,270 @@
+import logging
+import os
+from typing import List, Optional
+
```

Comment:
`from aind_slims_api.models import SlimsBehaviorSession, SlimsMouseContent, SlimsInstrument, SlimsUser`
instead of using inline imports that dont seem to be recognized by vscode. This probably happens because the inner import is using `__all__` syntax